Incorporate SignalrClient with AnyConnect Platform libraries

seym45 opened the pull request:
- Added build script for Android/RPi/Ubuntu
- Added necessary flags to CMake files

seym45 committed:
Incorporate SignalrClient with AnyConnect
Platform libraries

- Added build script for Android/RPi/Ubuntu
- Added necessary flags to CMake files

seym45 committed:
Incorporate SignalrClient with AnyConnect
Platform libraries

- Checked if andorid ndk path is valid or not.
- Generated signalrclient lib changed to static
  one from shared

KhaledMahmood commented on build.RPi.sh:
```diff
@@ -0,0 +1,47 @@
+CPPRESDSDK_ROOT=${PLATFORM_ROOT}/accessapi-artifacts-RPi-ms/cpprest
+OPENSSL_ROOT=${PLATFORM_ROOT}/openssl-1.0.2l-RPi
+
+command="cmake .. -DCMAKE_TOOLCHAIN_FILE=$HOME/projects/.launchers/automation/build/RPi/RPi.toolchain  \
```
Please use the path "/opt/RPi/RPi.toolchain" instread of "$HOME/projects/.launchers/automation/build/RPi/RPi.toolchain". I guess /opt/ is the actual directory for all toolchains.

seym45 replied:
Changed accordingly.

KhaledMahmood reviewed (changes requested):
Please consider changing the toolchain path for RPi

seym45 committed:
Change RPi toolchan path

KhaledMahmood reviewed (approved):
Looks good. Approved.

seym45 committed:
Fix Cmake files to build for iOS

seym45 committed:
Incorporate signalr with iOS

-As SignalR is dependent on CPPRest, Commented out few preprocessor
directives to build using xcodebuild.

KhaledMahmood reviewed (approved):
Looks good.

KhaledMahmood reviewed (approved):
Looks good.
